src/search.cpp

```diff
@@ -1612,7 +1608,7 @@ namespace {
     PieceType captured = type_of(pos.piece_on(to_sq(bestMove)));
 
     bonus1 = stat_bonus(depth + 1);
-    bonus2 = pastPV || bestValue > beta + PawnValueMg ? bonus1               // larger bonus
+    bonus2 = bestValue > beta + PawnValueMg ? bonus1               // larger bonus
                                                       : stat_bonus(depth);   // smaller bonus
```

Comment:
Can you fix the alignment?

Reply:
I'm not sure my git skills are up to doing that. This is a branch in my repo, but it's following (tracking?) Joost's branch. I can make another commit, but I don't know if I can then squash and push that since f10498b is his commit, not mine. Easier to fix it later in one of the trivial change patches I would say.